dev/build_release.py

```diff
@@ -211,28 +212,62 @@ def determine_gradle_root(self, name):
       gradle_root = name if name != 'spinnaker' else self.__project_dir
       return gradle_root
 
-  def start_build_target(self, name, target):
-      """Start a subprocess to build the designated target.
+  def start_candidate_release(self, name):
```

Comment:
Ok, I can rename the function but unfortunately the gradle task that is executed has to be either `candidate` or `final`.